.github/workflows/runtimer.yml

```diff
@@ -0,0 +1,55 @@
+name: Runtimer lets you bake a custom JRE on GH Actions
+
+# Controls when the workflow will run
+on:
+  workflow_call:
+    inputs:
+      java-version:
+        description: 'JVM version'
+        required: false
+        type: string
+        default: '11'
+      java-distribution:
+        description: 'JVM distribution'
+        required: false
+        type: string
+        default: 'temurin'
+      java-modules:
+        description: 'Java Modules'
+        required: false
+        type: string
+        default: 'java.base,java.logging,java.naming,java.xml,java.sql,java.transaction.xa,java.security.sasl,java.management'
```

Comment:
Maybe the default should be restricted to java.base since obviously not every project will require all of these modules. (The release.yml inputs should stay as-is for lrsql compatibility, of course.)

Reply:
will clojure itself work with only java.base? Why make the defaults break stuff? In addition, this is all academic until `lrsql` is migrated away from depending on the released artifacts this makes.

Reply:
Sorry, I should've said "the minimum that is required to run Clojure" (which may be tricky to determine because of runtime classloading).

As for lrsql, I actually don't think it'll matter since it downloads fixed artifacts from a tag anyways. So removing or even adding modules here shouldn't matter for it.

Reply:
I think the collection of modules required by lrsql is a decent starter point for a lot of projects, it's close to the kitchen sink without undue beanage, etc. If people want to pare down, the input lets them do that.

Reply:
Sure, though we should encourage users pare down their modules as much as possible. We can revisit this if we notice certain packages get removed all the time.

Reply:
sounds good